dvc/remote/local/__init__.py

```diff
@@ -251,6 +171,41 @@ def move(self, from_info, to_info):
 
         move(from_info, to_info, mode=mode)
 
+    def copy(self, from_info, to_info):
+        shutil.copyfile(from_info.fspath, to_info.fspath)
+
+    def symlink(self, from_info, to_info):
+        System.symlink(from_info.fspath, to_info.fspath)
```

Comment:
`System.symlink()` works with Path-like objects, so pass them as is. The overall approach is:
- our utilities accept Path-like
- `os` utilities only accept them in Python 3.6+, so we wrap into `fspath_py35()`
- if we want to convert to string we use stringification

So `.fspath` should never be used outside `PathInfo` implementation. We don't always follow these rules in current code, but new code should follow.

Reply:
@Suor , it would be great to have a doc to put down code conventions